dubbo-serialization/dubbo-serialization-hessian2/src/main/java/org/apache/dubbo/common/serialize/hessian2/Hessian2SerializerFactory.java

```diff
@@ -16,12 +16,27 @@
  */
 package org.apache.dubbo.common.serialize.hessian2;
 
+import com.alibaba.com.caucho.hessian.io.Deserializer;
+import com.alibaba.com.caucho.hessian.io.HessianProtocolException;
 import com.alibaba.com.caucho.hessian.io.SerializerFactory;
+import org.apache.dubbo.common.logger.Logger;
+import org.apache.dubbo.common.logger.LoggerFactory;
+import org.apache.dubbo.common.utils.ConcurrentHashSet;
+import org.apache.dubbo.common.utils.StringUtils;
+
+import java.util.Set;
 
 public class Hessian2SerializerFactory extends SerializerFactory {
+    private final static Logger logger = LoggerFactory.getLogger(Hessian2SerializerFactory.class);
 
     public static final SerializerFactory SERIALIZER_FACTORY = new Hessian2SerializerFactory();
 
+    /**
+     * For those classes are unknown in current classloader, record them in this set to avoid
+     * frequently class loading and to reduce performance overhead.
+     */
+    private Set<String> typeNotFoundDeserializer = new ConcurrentHashSet<String>(8);
```

Comment:
You can use the dimond operator.

private Set<String> typeNotFoundDeserializer = new ConcurrentHashSet<>(8);

Reply:
Would you care explain us about using 8 as size for ConcurrentHashSet?